iggtools/subcommands/midas_run_genes.py

```diff
@@ -205,7 +173,7 @@ def normalize(genes, covered_genes, markers):
     # compute median marker depth for each species
     species_markers_coverage = defaultdict(float)
     for species_id, marker_depths in species_markers.items():
-        species_markers_coverage[species_id] = np.median(list(marker_depths.values()))   # np.median doesn't take iterators
+        species_markers_coverage[species_id] = np.median(list(marker_depths.values())) # np.median doesn't take iterators
```

Comment:
This introduces a lint whitespace formatting bug.  Two spaces are always required before '#' at end of line.